src/main/java/uk/gov/ida/matchingserviceadapter/MatchingServiceAdapterModule.java

```diff
@@ -304,8 +305,8 @@ public String getHubConnectorEntityId(MatchingServiceAdapterConfiguration config
     @Provides
     @Singleton
     @Named("AcceptableHubConnectorEntityIds")
-    public String[] getAcceptableHubConnectorEntityIds(MatchingServiceAdapterConfiguration configuration) {
-        return configuration.isEidasEnabled() ? configuration.getEuropeanIdentity().getAcceptableHubConnectorEntityIds() : new String[0];
+    public List<String> getAcceptableHubConnectorEntityIds(MatchingServiceAdapterConfiguration configuration) {
+        return configuration.isEidasEnabled() ? configuration.getEuropeanIdentity().getAcceptableHubConnectorEntityIds(configuration.getMetadataEnvironment()) : new LinkedList<>();
```

Comment:
Do we have to pass the `configuration.getMetadataEnvironment()` value? Can it be retrieved within the `getAcceptableHubConnectorEntityIds()` method itself?

Reply:
I don't believe it can.

Reply:
Fine